blockchain/validate_test.go

```diff
@@ -111,8 +111,7 @@ func TestCheckBlockSanity(t *testing.T) {
 func TestCheckSerializedHeight(t *testing.T) {
 	// Create an empty coinbase template to be used in the tests below.
 	coinbaseOutpoint := wire.NewOutPoint(&chainhash.Hash{}, math.MaxUint32)
-	coinbaseTx := wire.NewMsgTx()
-	coinbaseTx.Version = 2
+	coinbaseTx := wire.NewMsgTx(1)
```

Comment:
Yes, it should've been version 1 all along.  The serialized height in the coinbase is required as of version 2 _blocks_ as opposed to transactions.  Version 2 transactions did not exist at that point in time.